blobmodel/stochasticality.py

```diff
@@ -43,11 +43,15 @@ def __init__(
         wy_dist: str = "deg",
         vx_dist: str = "deg",
         vy_dist: str = "normal",
+        spx_dist: str = "deg",
+        spy_dist: str = "deg",
         A_parameter: float = 1.0,
         wx_parameter: float = 1.0,
         wy_parameter: float = 1.0,
         vx_parameter: float = 1.0,
         vy_parameter: float = 1.0,
+        spx_parameter: float = 0.5,
```

Comment:
This might be personal taste, but I would have used a longer name such as shape_parameters_x instead of spx. One could say the same thing about wx and vx, so I leave that to you.

Reply:
I changed it, but becomes a bit lol now as "shape_param_x_parameter", since it is a parameter that models the distribution of shape parameters XD